src/core/src/runtime/itensor.cpp

```diff
@@ -160,7 +160,17 @@ void ITensor::copy_to(const std::shared_ptr<ov::ITensor>& dst) const {
 
     bool finish = false;
     for (size_t dst_idx = 0, src_idx = 0; !finish;) {
-        memcpy(dst_data + dst_idx, src_data + src_idx, src_strides[src_strides.size() - 1]);
+        if (get_element_type() == element::string) {
```

Comment:
For future:
maybe we need to add a flag to our types to denote they are not trivially copyable?
In this case all such types need to have explicit copying and avoid memcpy

Reply:
Agree, let us do it.